src/file/file.ts

```diff
@@ -268,3 +269,6 @@ export const createReadStream = async (filePath: string) =>
 export const testFileRegExp = /\.test\.(\d+\.)?sas$/i
 
 export const isTestFile = (fileName: string) => testFileRegExp.test(fileName)
+
+export const getLineEnding = (content: string) =>
```

Comment:
a comment here about how this works would be great

Reply:
done